src/managers/epicenter-channel-manager.js

```diff
@@ -222,6 +251,25 @@ var EpicenterChannelManager = classFrom(ChannelManager, {
         return channel;
     },
 
+    /**
+     * Create and return a publish/subscribe channel (from the underlying [Channel Manager](../channel-manager/)) for the given collection. (The collection name is specified in the `root` argument when the [Data Service](../data-api-service/) is instantiated.) Must be one of the collections in this account (team) and project.
+     *
+     * There are automatic notifications from Epicenter on this channel when data is created, updated, or deleted in this collection. See more on [automatic messages to the data channel](../../../rest_apis/multiplayer/channel/#data-messages).
+     *
+     * **Example**
+     *
+     *     var cm = new F.manager.ChannelManager();
+     *     var gc = cm.getDataChannel('survey-responses');
+     *     gc.subscribe('', function(data) { console.log(data); });
```

Comment:
The second parameter is a `meta` - see comment in https://issues.forio.com/browse/WHARTONART-38 for specifics